c++/CMakeLists.txt

```diff
@@ -82,11 +82,40 @@ else()
 endif()
 
 # Source =======================================================================
-
+include(CapnProtoMacros)
 add_subdirectory(src)
 
 # Install ======================================================================
 
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+  "${CMAKE_CURRENT_BINARY_DIR}/CapnProtoConfigVersion.cmake"
+  VERSION ${VERSION}
+  COMPATIBILITY AnyNewerVersion
+)
+set(ConfigPackageLocation ${LIB_INSTALL_DIR}/cmake/CapnProto)
+
+configure_package_config_file(cmake/CapnProtoConfig.cmake.in
+  ${CMAKE_CURRENT_BINARY_DIR}/CapnProtoConfig.cmake
+  INSTALL_DESTINATION ${ConfigPackageLocation}
+  PATH_VARS INCLUDE_INSTALL_DIR
+)
+export(EXPORT CapnProtoTargets
+  FILE "${CMAKE_CURRENT_BINARY_DIR}/CapnProtoTargets.cmake"
+  NAMESPACE CapnProto::
```

Comment:
Most other config modules I have come across use upper case case names (e.g. Qt5::Core, GLEW::GLEW KF5::Config, etc). Also the Cmake docs use "Upstream::", so I guess it makes sense to be consistent with other packages.